terraform/modules/jvs-services/jvs-ui.tf

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "ui_cloud_run" {
+  source = "git::https://github.com/abcxyz/terraform-modules.git//modules/cloud_run?ref=5445543e21491176528fb5cd7adcb505d9dec5dd"
+
+  project_id            = var.project_id
+  region                = var.region
+  name                  = var.jvs_ui_service_name
+  image                 = var.jvs_ui_service_image
+  ingress               = var.service_ingress
```

Comment:
@shan4993 By default, I'm using `all` ingress assuming IAP is not in the picture. I suppose the future integration test can carry out without IAP.

In the e2e setup, I plan to introduce gclb and IAP https://github.com/abcxyz/jvs/issues/194.

Reply:
Sounds good. The future integ test would need that email header from IAP but since we arent relying on the JWT header i imagine we can set a random value via integ test.

Reply:
Yes, that's what I was thinking.